addons/common/fnc_addMagazine.sqf

```diff
@@ -49,7 +49,7 @@ if (!isClass _config || {getNumber (_config >> "scope") < 2}) exitWith {
 };
 
 if (_verify) then {
-    if (_unit canAdd _item) then {
+    if (_unit canAdd _item || {[_unit, _item, 1, true, true, true] call CBA_fnc_canAddItem}) then { // canAdd works for vehicles and units, CBA_fnc_canAddItem is more precise for units, but only try if canAdd fails for them
```

Comment:
Wouldn't inline a comment this long.

> CBA_fnc_canAddItem is more precise for units, but only try if canAdd fails for them

I have more questions now. What does `precise` mean in this context?

Reply:
`canAdd` looks at total unit load and can return false for units when they do actually have space in a container.
`CBA_fnc_canAddItem` just checks each container directly and returns true if there is room.

I'd like to only use `CBA_fnc_canAddItem`, but `CBA_fnc_addMagazine` can take a vehicle as input, so we have to use `canAdd` as well for them.

Reply:
Okay, so `canAdd` implies `canAddItem` for units.

Shouldn't we rewrite `canAddItem` to work on vehicles as well and drop `canAdd` entirely for this?